client/src/pages/Services/style.js

```diff
@@ -0,0 +1,39 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    margin: theme.spacing(3, 4, 3),
+    width: '100%',
+  },
+  title: {
+    margin: theme.spacing(6, 0, 0),
+  },
+  container: {
+    display: 'flex',
+    [theme.breakpoints.down('sm')]: {
+      flexDirection: 'column',
+    },
+    justifyContent: 'flex-start',
+    margin: theme.spacing(4, 0, 6),
+  },
```

Comment:
if you have , this section need improvement to be more responsive
![image](https://user-images.githubusercontent.com/32321796/114786462-d33b7600-9d86-11eb-93fb-ac270edc5ed0.png)